lib/connextclient/ConnextClient.ts

```diff
@@ -58,7 +59,7 @@ interface ConnextClient {
 const waitForPreimageByHash = (
   client: SwapClient,
   expectedHash: string,
-  errorTimeout = 75000,
+  errorTimeout = 34000,
```

Comment:
Is this supposed to be 35000? Or did you intentionally make it slightly lower than the swap timeout?

Reply:
Yep, this is the timeout for `ConnextClient.sendPayment` which should error before the global swap timeout. I think this is necessary because otherwise the subscription of observable is never cleaned up.